R/aggreger_ki_prop.R

```diff
@@ -148,3 +148,5 @@ aggreger_ki_prop = function(d_ki_ind, alfa = 0.05) {
 
   d_sammendrag
 }
+
+#Kommentar som ikkje fylgjer kodestilen
```

Comment:
Desse må vekk.

Reply:
Ja, dei er berre med for å få testa den automatiske sjekken.

Reply:
Dei trengst vel ikkje for å testa det? Styler vert jo køyrt på alle filene, ikkje berre dei endra. Og me har mykje som ikkje følgjer styler-formateringa.

Reply:
Ja, men sjølve GitHub-actionen vert berre køyrt viss filer med R-kode er endra:
```
on:
  pull_request:
    branches: [main]
    paths: ["**.[rR]", "**.[qrR]md", "**.[rR]markdown", "**.[rR]nw", "**.[rR]profile"]
```

Reply:
Å! Okei. Men du må fjerna det viss eg skal godkjenna PR-en. Det er det einaste som manglar no.

Reply:
Fjerna (og rebasert på main).